src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -110,10 +111,13 @@ function OptionRowLHNData({
     }, [fullReport, linkedTransaction, reportActions, personalDetails, preferredLocale, policy, parentReportAction, transaction]);
 
     useEffect(() => {
-        if (!optionItem || optionItem.hasDraftComment || !comment || comment.length <= 0 || isFocused) {
+        const draftReportIDs = DraftReportUtils.getInstance().getDraftReportIDs();
```

Comment:
In general I am a bit sceptical about the naming as I feel like this can be a bit misleading once we bring draft reports (open state reports) to the app. However, nothing clean comes to my mind right now about alternative name

Reply:
`draftCommentReportIDs`